prompting/settings.py

```diff
@@ -16,13 +16,17 @@
 
 # Bittensor
 
-NETUID = bt_config.netuid or int(os.environ.get("NETUID"))
+NETUID = bt_config.netuid or int(os.environ.get("NET_UID"))
 TEST = NETUID != 1
 OPENAI_API_KEY = os.environ.get("OPENAI_API_KEY")
 WALLET_NAME = bt_config.wallet.name or os.environ.get("WALLET_NAME")
 HOTKEY = bt_config.wallet.hotkey or os.environ.get("HOTKEY")
 AXON_PORT = bt_config.axon.port or int(os.environ.get("AXON_PORT"))
-ORGANIC_WHITELIST_HOTKEY = os.environ.get("ORGANIC_WHITELIST_HOTKEY")  # Replace with the actual default value
+ORGANIC_WHITELIST_HOTKEY = os.environ.get(
+    "ORGANIC_WHITELIST_HOTKEY",
+    # OTF hotkey.
+    "5F4tQyWrhfGVcNhoqeiNsR6KjD4wMZ2kfhLj4oHYuyHbZAc3"
```

Comment:
can we include this hotkey safely here or is there any security risk? I know it should be fine in principle - just double-checking!

Also, should this only be a single hotkey or should we include hotkeys from all relevant validators?

Reply:
We can include it, since the hotkey is visible over the network. Multiple hotkeys would be nice future enhancement, but for now we use only one